core/src/ast_builder/expr/function.rs

```diff
@@ -63,6 +78,17 @@ mod tests {
         test_expr(actual, expected);
     }
 
+    #[test]
+    fn function_ifnull() {
+        let actual = col("updated_at").ifnull(col("created_at"));
+        let expected = "IFNULL(updated_at, created_at)";
+        test_expr(actual, expected);
+
+        let actual = text("HELLO").ifnull(text("WORLD"));
+        let expected = "IFNULL('HELLO', 'WORLD')";
+        test_expr(actual, expected);
```

Comment:
Current two unit tests are identical.
Both tests use
```rust
pub fn ifnull(self, another: ExprNode) -> ExprNode {
   ..
}
```

Let's replace one of those to use
```rust
pub fn ifnull<T: Into<ExprNode>, V: Into<ExprNode>>(expr: T, then: V) -> ExprNode {
   ..
}
```

Reply:
You are right. I'll fix one of them.